code/modules/shuttle/dock_generator.dm

```diff
@@ -0,0 +1,102 @@
+/obj/item/whiteship_port_generator
+	name = "docking area signaller"
+	desc = "A signaling device for the NT expeditionary vessel, used to designate new docking areas."
+	icon = 'icons/obj/device.dmi'
+	icon_state = "gangtool-white"
+
+	var/list/placed_docks = list()
+	var/max_docks = 3
+
+/obj/item/whiteship_port_generator/examine(mob/user)
+	. = ..()
+	var/plural = length(placed_docks) > 1
+	. += "There [plural ? "are" : "is"] [max_docks - length(placed_docks)] use[plural ? "s" : ""] left."
```

Comment:
![image](https://github.com/ParadiseSS13/Paradise/assets/77684085/d866c983-cb50-42c7-9c9f-63324b51aa70)

var/plural should be checking for "max_docks - length(placed_docks)" and not only "length(placed_docks)" right?

Reply:
Yup, fixed.